src/BookRender.hs

```diff
@@ -1,27 +1,27 @@
 module BookRender where
 
-import Numeric
+import Control.Monad
+import Data.FileEmbed
+import Data.Foldable
+import Data.Generics.Uniplate.Data
+import qualified Data.List as List
+import qualified Data.Map as Map
+import Data.Monoid
+import Data.Reflection
+import Data.String
 import Data.Text (Text)
 import qualified Data.Text as Text
 import qualified Data.Text.Lazy as Text (toStrict)
 import Lens.Micro.Platform
-import qualified Data.Map as Map
 import Network.URI
-import Data.Monoid
-import Data.String
-import Data.Foldable
-import qualified Data.List as List
-import Data.Reflection
-import Data.Generics.Uniplate.Data
-import Control.Monad
-import Data.FileEmbed
+import Numeric
 
 import Text.Blaze.Html.Renderer.String (renderHtml)
 import Text.Blaze.Html5 ((!))
 import qualified Text.Blaze.Html5 as H
 import qualified Text.Blaze.Html5.Attributes as A
 
-import Clay ((?), (|>), (#))
+import Clay (( # ), (?), (|>))
```

Comment:
why is `( # )` with spaces but `(?)` without?

Reply:
Bug in `stylish-haskell`. Can't handle operators started with `#` properly :disappointed: